.github/workflows/cmake.yml

```diff
@@ -144,13 +216,16 @@ jobs:
             libxxhash-dev \
             libzstd-dev \
             mecab-naist-jdic \
-            ninja-build \
             rapidjson-dev \
             ruby-dev \
             zlib1g-dev
       - name: CMake
         run: |
           ccache --show-stats --verbose --version || :
+          CMAKE_ARGS=""
+          if [ "${APACHE_ARROW}" = "main" ]; then
+            CMAKE_ARGS="${CMAKE_ARGS} -DArrow_DIR=$PWD/apache-arrow/install/lib/cmake/Arrow"
+          fi
```

Comment:
Can we remove this by using the same install directory for Apache arrow and Groonga?